### PR TITLE
added the sendAction method

### PR DESCRIPTION
**Pull request by kalepail**
https://developers.facebook.com/docs/messenger-platform/send-api-reference/sender-actions

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -6,6 +6,10 @@ class FBMessenger {
     this.notificationType = notificationType || 'REGULAR'
   }
 
+  sendAction (id, action) {
+    this.sendMessage(id, action, null, null);
```
Comment (charlesaraya) [inline src/index.js]: There's no true need to pass 'null' values, just omit them.

File: src/index.js [hunk 2]
```diff
@@ -84,17 +88,25 @@ class FBMessenger {
       cb = notificationType
       notificationType = this.notificationType
     }
+
+    const json = {
+      recipient: {
+        id: id
+      }
+    }
```
[the remark this answers is not in the document]
Comment (charlesaraya) [reply src/index.js:96]: not need in this case. [Automatic Semicolon Insertion](http://www.ecma-international.org/ecma-262/6.0/#sec-automatic-semicolon-insertion)

File: src/index.js [hunk 3]
```diff
@@ -84,17 +88,25 @@ class FBMessenger {
...
+
+    if (typeof data === 'string') {
+      json.sender_actions = data;
```
[the remark this answers is not in the document]
Comment (charlesaraya) [reply src/index.js]: This field must be written that way, and all the ones you point out below. Check the Send API for more information.

```
/* Send Request to set sender action to 'typing on'*/
curl -X POST -H "Content-Type: application/json" -d '{
  "recipient":{
    "id":"USER_ID"
  },
  "sender_action":"typing_on"
}' "https://graph.facebook.com/v2.6/me/messages?access_token=PAGE_ACCESS_TOKEN"
```

**Comment by DiegoRBaquero** [comment]
Hey @tyvdh . I'm sorry I have been so busy to check this PR. I have given you access to this repo and NPM publish to the package so you can merge this and publish it.

I appreciate your work and trust you.

**Pull request merged**